db/schema.rb

```diff
@@ -52,6 +52,8 @@
     t.datetime "updated_at", precision: 6, null: false
     t.integer "employee_id", null: false
     t.integer "receiver_id", null: false
+    t.bigint "company_value_id", null: false
```

Comment:
I am not sure, why there is null: false here, if there is null: true in the migration - any ideas what might be the cause of this?

Reply:
Yes, I had a little conflict between migrations, I tried to fix it ineptly by hand, but eventually I rolled back the penultimate migration and set it up again so that all parameters had the same constraints.